FWUV-258: Persist License together with HmacID (#102)
FWU-DE/fwu-kc-extensions: remove-user-on-logout/README.md

This extension is to make sure the user is removed from keycloak on the user logout and session expiration.

> Moreover, with the recent development before removing the user the licence associated with the user is released.
> Also the licence that is stored in the database together with the hmacId is deleted.

Reviewer: ```suggestion
Moreover, with the recent development before removing the user the licence associated with the user is released in Licence Connect and deleted from the Keycloak database.
```
